src/app/components/no-page/no-page.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="content">
+    <h1>
+        Looks like there is no page with this url.
+    </h1>
+    You can start navigate from
+    <a href="http://localhost:4200">
```

Comment:
It's better to use [routerLink] attribute. By this way you would prevent "hard" reloading of your application

Reply:
Changed